earthly/mdlint/Readme.md

```diff
@@ -0,0 +1,47 @@
+# Markdown Linter
+
+This Earthly Target and UDC enables uniform linting of Markdown files to maintain consistency and quality.
+
+## How it works
+
+Linting is performed with the [`mdlint-cli2`](https://github.com/DavidAnson/markdownlint-cli2) program.
+
+Use the `MDLINT` Earthly UDC to enable uniform and consistent Markdown Format checking.
+
+This linter is to be used in preference to any other Markdown linter.
+This is because we need to provide uniform and consistent Markdown formatting and linting across the project and between projects.
+
+## Invocation
+
+In an Earthfile in your repo, add the following:
+
+```earthfile
+markdown-lint:
+    # Check Markdown in this repo.
+    LOCALLY
```

Comment:
See previous comment about recommending LOCALLY here.

Reply:
Same reply.  its not running inside a container build.